http_geocoder.go

```diff
@@ -131,11 +131,13 @@ func response(ctx context.Context, url string, obj ResponseParser) error {
 	}
 
 	body := strings.Trim(string(data), " []")
+	DebugLogger.Printf("Received response: %s\n", body)
 	if body == "" {
 		return nil
 	}
 	if err := json.Unmarshal([]byte(body), obj); err != nil {
-		Logger.Printf("payload: %s\n", body)
+		ErrLogger.Printf("Error unmarshalling response: %s\n", err.Error())
+		ErrLogger.Printf("payload: %s\n", body)
```

Comment:
the StdLogger interface is a bit overkill; i'm reluctant to add more implementations onto it.
however i do see the point of adding debug level log in this case.
how about combining the 2 error logs here into one?

Reply:
I'm ok to combine two error logs into single one, but what's your thoughts regarding addition of debug logging?
As an alternative to adding 2nd implementation of StdLogger interface for debug purposes we could modify or extend the interface itself by adding distinct methods for different log levels.

Reply:
i have no problem with logging, but the StdLogger abstraction doesn't have much value to me. i'm talking about long before this pull request.
we don't have much logging in the code, otherwise, i'd go for a proper structured logging.

Reply:
Any recommendations for changing this pull request or we should close it as not accepted?
I'll keep the changes in my fork then since I need logging with debug level.

Reply:
sorry if i didn't made myself clear enough.
> how about combining the 2 error logs here into one?

Reply:
[combined two err log messages into single one](https://github.com/codingsince1985/geo-golang/pull/75/commits/41440748157b5108c9e98ceaba73a949c0e33c95)
Aleksey Korolev committed 22 minutes ago

[removed logging of response body with err log level since i…](https://github.com/codingsince1985/geo-golang/pull/75/commits/5acb32a5e430316c8dcd1e50ff69ec4e921b85c5) 
Aleksey Korolev committed 19 minutes ago